components/commons/KnitNavbar.vue

```diff
@@ -39,7 +39,7 @@ export default {
         },
         {
           text: 'Artykuły',
-          path: '/articles'
+          path: '/wszystkie'
```

Comment:
I don't really like Polish name in path R W sure we wanna do this?

Reply:
I totally agree with this. `Wszystkie` sounds really, really weird.

Reply:
So how would you name all articles in path?
`/wszystkie` seems good to me

Reply:
`wszystkie` simulates the category while articles does not

Reply:
The perfect solution for me would be `/articles` for all articles and '/articles/:category' for a specific category. Is it possible? If not, i suggest `/articles/all`

Reply:
@zrebcu411 for me that's perfect

Reply:
I would disagree with that. We should prepare friendly urls because of search engines. From my experience with content products the best option is always present the url like `/categoryCode/slug/shortid`

Reply:
no problem but we'll have `projects` right?

Reply:
@k911 supporting two langs in a content targeted site is not possible I think. Whole content will be created by KNIT members, and we cannot tell them to write articles, offers etc. in two languages. On the other hand, if we'll be translating only specific parts of site e.g. menu, the site will be not consistent.

Reply:
As far as I know we do not want to create multi language site. Imagine how hard we would have to work to mantain multilanguage content site. 

Look how others do content services.
https://moz.com/learn/seo/url
https://wiadomosci.wp.pl/kaczynski-doszlismy-do-celu-kolejne-marsze-beda-raz-na-rok-6239808818247809v
http://www.blikk.hu/aktualis/politika/igy-vezetett-vona-gabor-utja-a-lemondasig/29q9w97
https://www.theguardian.com/world/europe-news
https://www.washingtonpost.com/business/economy/for-mark-zuckerberg-protecting-facebook-means-stepping-into-the-spotlight/2018/04/09/31518554-3c0a-11e8-a7d1-e4efec6389f0_story.html?hpid=hp_hp-top-table-main_zuckerberg-240p%3Ahomepage%2Fstory&noredirect=on

Imagine what users would feel when they see url like
 `baseurl/articles/category/slug/id`

It is possible that the slug would not be visible which is not something we would like to have.

Reply:
Events, projects etc are not the main purpose of our site. The main purpose is of course presenting the articles. Others might have path startig from events, projects and so on.

Reply:
@FieryCod all examples you presented relate to the single resource, not list

Reply:
There sereval options to resolve that issue (e.g. like wikipedia, we could mark an article in a future if it is translated, so that's no problem)
i18n != supporting one optional language
All I'm saying is that we should not block possibility in future, or result in complicated refactor if we decide to.

Reply:
@zrebcu411 remove slug and then everything is list

Reply:
@k911 We will not support ever multilanguage website. Services with articles are in one main language. So we might switch to english in whole site or just use Polish.

Reply:
@FieryCod @zrebcu411 Have you ever been to wikipedia? Why we should block opportunity to write article in english? What if somebody wrote article even in chineese? 😄

Reply:
And remeber that KNIT is not all about news content, it is supposed to be a platform, however news content is our primary goal

Reply:
Wikipedia does not have articles I believe and it's whole idea is to present bare informations.

Reply:
Wikipedia is a specific site, it is a worldwide encyclopedia. Our site aims to connect specific group of people

Reply:
Yup totally agree with you @zrebcu411

Reply:
@FieryCod show me an example of a site with a list of ALL articles and separate smaller sublists. In all sites, a list of all articles is on mainpage

Reply:
We lost the point, do you really like polish name in url? @FieryCod @zrebcu411?

Reply:
@zrebcu411 That's not true! On mainpage they got only selected subset of articles. That's true that they does not have special category like `wszystkie` simply because they got so many articles that they do not need it.